.github/workflows/docker.yml

```
@@ -38,9 +42,22 @@ jobs:
           echo "NEXT_PUBLIC_WEB3_CLIENT_ID=${{ secrets.NEXT_PUBLIC_WEB3_CLIENT_ID }}" >> .env
           cat .env
 
+      - name: Get the Git tag
+        id: get_tag
+        run: echo "GIT_TAG=$(echo ${GITHUB_REF#refs/tags/})" >> $GITHUB_ENV
```

actionlint (1.7.3)

shellcheck reported issue in this script: SC2116:style:1:15: Useless echo? Instead of 'cmd $(echo foo)', just use 'cmd foo'

(shellcheck)